zebra/rt_netlink.c

```diff
@@ -2361,6 +2361,7 @@ ssize_t netlink_route_multipath_msg_encode(int cmd, struct zebra_dplane_ctx *ctx
 	}
 
 	if ((!fpm && kernel_nexthops_supported()
+	     && (p->family != AF_INET6 || !src_p)
```

Comment:
Shouldn't we tell the operator about such a thing? Because now we just hide the error/warning being displayed from the kernel, right?

Reply:
Not necessary. Instead of using nexthop object, just send RTNL message with plain nexthops, as when `no zebra nexthop kernel enable` is configured. Kernel is happy with this.

Reply:
I think adding a bit of context in the nexthop_groups.rst file to explain this should be sufficient.

Reply:
> I think adding a bit of context in the nexthop_groups.rst file to explain this should be sufficient.

nexthop_groups.rst is about `nexthop-group` configured by user, which is a different thing from kernel nexthop objects. Kernel nexthop is sort of internal implementation of zebra. Is it more suitable to put it in zebra.rst?

Reply:
sure put it in zebra.rst.  I want this documented.